src/site/_includes/components/component-preview/macro.njk

```diff
@@ -10,6 +10,7 @@
 {%- endif -%}
 
 {%- set idPrefix = params.componentName + "--" + params.exampleName + idSuffix -%}
+{%- set iframe = params.iframe | default(true) -%}
 
 <div class="guidance-component-preview js-guidance-component-preview">
   <div class="guidance-component-preview__toolbar">
```

Comment:
The href on the next line needs updating to add `/components/{{ params.componentName }}/` (to fix the error message open in a new tab link).